mautic/mautic
# Add description on Tag form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



This PR adds summary and description on Tag form so the marketers can describe more precisely what given tag represents.

Note: We are now able to edit tag only the first time we open the form. If we want to rename the tag we have to create a new one, this however is not true for summary and description, we can update those two fields as much as we want (this is the expected behavior, you can read about reasoning behind this decision here: #10927).

I will try to add tests (and probably need some assistance on what scenario do I actually add) if codecov goes down.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to new tag form
3. You can now add description to tag.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10799"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
